cinema/urls.py

```diff
@@ -1,10 +1,35 @@
-from django.urls import path
+from django.urls import path, include
+from rest_framework import routers
 
-from cinema.views import movie_list, movie_detail
+from cinema.views import (
+    GenreList,
+    GenreDetail,
+    ActorList,
+    ActorDetail,
+    CinemaHallViewSet,
+    MovieViewSet
+)
+
+router = routers.DefaultRouter()
+router.register("movies", MovieViewSet)
+
+hall_list = CinemaHallViewSet.as_view(
+    actions={"get": "list", "post": "create"})
```

Comment:
```suggestion
    actions={"get": "list", "post": "create"}
    )
```

Also here fix indentation and in other similar cases